nfadeeva/assignment4/count_islands.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+
+def neighbors(input_array, num_rows, num_cols, x, y):
```

Comment:
Maybe for clarity rename input_array to map?

Reply:
I fixed all places and renamed input_array to tiles because map is the built-in name in python)